packages/lb-annotation/src/core/toolOperation/pointCloud2dOperation.ts

```diff
@@ -143,6 +160,32 @@ class PointCloud2dOperation extends PolygonOperation {
     return PolygonUtils.getPolygonByIDs(this.polygonList, this.selectedIDs);
   }
 
+  public updateSelectedPolygonsPoints(offset: Partial<ICoordinate>) {
+    if (this.selectedPolygons && this.selectedPolygons?.length > 0) {
+      const originPolygonList = _.cloneDeep(this.selectedPolygons!);
+      const updateList: UpdatePolygonByDragList = [];
+
+      this.selectedPolygons?.forEach((polygon, index) => {
+        polygon.pointList = polygon.pointList.map((point) => {
```

Comment:
This way will change the origin polygonList in `this.polygonList`. Creating a new PointList is better.

Reply:
It need to be change pointList